Description: systemd: Use Type=simple
 Use Type=simple rather than Type=forking.  There is no need to use forking,
 as ntpd has the -n option to skip fork().
Forwarded: no
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2017-11-06
--- a/etc/ntpd.service
+++ b/etc/ntpd.service
@@ -8,11 +8,9 @@
 Conflicts=systemd-timesyncd.service
 
 [Service]
-Type=forking
 PrivateTmp=true
-ExecStart=/usr/bin/ntpd -g -u ntp:ntp
+ExecStart=/usr/bin/ntpd -n -g -u ntp:ntp
 Restart=always
-RemainAfterExit=yes
 
 [Install]
 WantedBy=multi-user.target
